src/main/windows/windowManager.ts

```diff
@@ -484,20 +491,39 @@ export function selectPreviousTab() {
     }
 
     const currentTeamTabs = status.config?.teams.find((team) => team.name === currentView.tab.server.name)?.tabs;
+    const filteredTabs = currentTeamTabs?.filter((tab) => !tab.isClosed);
     const currentTab = currentTeamTabs?.find((tab) => tab.name === currentView.tab.type);
-    if (!currentTeamTabs || !currentTab) {
+    if (!currentTeamTabs || !currentTab || !filteredTabs) {
         return;
     }
 
-    const currentOrder = currentTab.order;
-
     // js modulo operator returns a negative number if result is negative, so we have to ensure it's positive
-    const nextOrder = ((currentTeamTabs.length + (currentOrder - 1)) % currentTeamTabs.length);
-    const nextIndex = currentTeamTabs.findIndex((tab) => tab.order === nextOrder);
-    const newTab = currentTeamTabs[nextIndex];
+    let currentOrder = currentTab.order;
+    let nextIndex = -1;
+    while (nextIndex === -1) {
```

Comment:
same as above, not really  sure why do we need the while loop

Reply:
See above